Jypeli/Controls/Keyboard/Keyboard.cs

```diff
@@ -239,6 +310,141 @@ public bool IsAltDown()
             return CurrentState.IsKeyDown(Key.LeftAlt) || CurrentState.IsKeyDown(Key.RightAlt);
         }
 
+        
+        
+        /// <summary>
+        /// Kuuntelee useiden näppäinten painalluksia, voi ottaa myös parametreja vastaan.
+        /// </summary>
+        /// <param name="k">Näppäimet, nämä voit antaa taulukkona, listana tai minä tahansa muuna IEnumerable rajapinnan toteuttavana tyyppinä</param>
+        /// <param name="state">Näppäimen tila</param>
+        /// <param name="handler">Mitä tehdään</param>
+        /// <param name="helpText">Ohjeteksti</param>
+        /// /// <returns>Palauttaa kuuntelijan, HUOM sidottu ensimmäiseksi annettuun näppäimeen ei tarvitse poistaa kaikista</returns>
+        /// <exception cref="InvalidDataException"> Jos tietorakenne on tyhjä heitetään poikkeus</exception>
+        public Listener ListenMultiple(IEnumerable<Key> k, ButtonState state, Action handler, string helpText)
+        {
+            ChangePredicate<KeyboardState> rule = MakeTriggerRuleMultiple(k, state);
+            //Lisätään kuuntelija ensimmäiselle näppäimelle. 
+            //Tässä valitetaan possible multiple enumeration, mutta foreach silmukan pitäisi nollata se meidän puolesta
+            var enumerator = k.GetEnumerator();
+            if (!enumerator.MoveNext())
+            {
+                throw new InvalidDataException("No keys in data structure, possibly empty data structure?");
+            }
+            Key key =  enumerator.Current;
+            //Muistetaan poistaa enumeraattori
+            enumerator.Dispose();   
```

Comment:
Jos yllä olevan tekee, tämä yksinkertaistuu hauksi:


```suggestion
            var firstKey = keys[0];
```